specification/app/resource-manager/Microsoft.App/preview/2024-08-02-preview/CommonDefinitions.json

```diff
@@ -590,6 +590,18 @@
           "type": "integer",
           "default": 10
         },
+        "cooldownPeriod": {
```

Comment:
The examples need to be updated for the new properties.

Reply:
Updated examples for every Container App example with scale. In commit: https://github.com/Azure/azure-rest-api-specs/pull/30154/commits/d3a73a221d18a474a52515f11434662db96627fa